README.md

```diff
@@ -1 +1,47 @@
-# Riyaz-Portfolio
\ No newline at end of file
+# Riyaz-Portfolio
+
+## Built With
+
+HTML
+CSS
+
+#### Getting Started
+
+https://r-ahmed2022.github.io/Riyaz-Portfolio/
+
+#### Getting Started
+
+Instructions on how to access my project locally. To Access my project locally please follow the following instructions;
+
+Enter this url: https://github.com/r-ahmed2022/Riyaz-Portfolio in your web browser.
+Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+Select download Zip option from drop down menu.
+
+## Prerequisites
+
+-> Setup GitHub repository
+-> Install git
+-> Setup Linters
+. Lighthouse
+. Weblint
+. Stylelint
+
+Deployment
+Authors
```

Comment:
[OPTIONAL] I recommend you remove the `Deployment` header and add ## before the Authors header so it looks like an actual header and gives more visibility to your social media links.

Reply:
Done as advised